stackbrew.js

```diff
@@ -38,7 +38,7 @@ for (version of versions) {
   let maintenance = new Date(`${config[version].maintenance}T00:00:00.00`).getTime();
   let isCurrent = foundCurrent ? false : isNaN(lts) || lts >= now;
   foundCurrent = isCurrent || foundCurrent;
-  let isLTS = foundLTS ? false : (maintenance >= now) && (now >= lts);
+  let isLTS = foundLTS ? false : (now >= lts);
```

Comment:
I think this may cause issues once more than 1 release enters the LTS range. I think last time, I just fudged one of the dates in the version.json, then set it back after the window

Reply:
It shouldn't. Once it it finds one, it stops looking.

Reply:
The previous check made sure it wasn't in maintenance, and I don't think there is a guarantee on the order that it will read them. This is fine for now, but i'd probably revert this after the window has passed